docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst

```diff
@@ -67,18 +67,9 @@ Setting up an ``IdProvider``
 Launching the ``TokenManagerHandler``
 -------------------------------------
 
-Run the following commands from a DIRAC client to install the ``Framework/TokenManager`` Tornado service:
+Install the ``Framework/TokenManager`` Tornado service, following :ref:`these instructions <httpsTornado>`
```

Comment:
Better link here? https://dirac--7275.org.readthedocs.build/en/7275/DeveloperGuide/TornadoServices/index.html#install-a-service

Reply:
Do I need to add another label in target or is there some hidden magic that I don't know?

Reply:
You need to add a label.

cf. https://stackoverflow.com/a/5889200

I think I tried autosectionlabel, but there were too many clashes?